src/Reflection/Adapter/ReflectionMethod.php

```diff
@@ -273,7 +273,7 @@ public function isProtected()
      */
     public function isAbstract()
     {
-        return $this->betterReflectionMethod->isAbstract();
+        return $this->betterReflectionMethod->isAbstract() || $this->betterReflectionMethod->getDeclaringClass()->isInterface();
```

Comment:
@kukulich I think this should probable be fixed in `\Roave\BetterReflection\Reflection\ReflectionClass::isAbstract` too, perhaps?

Reply:
It looks the current version is right: https://3v4l.org/EBjjL

Reply:
wow, PHP is confusing 🤣

Reply:
Oh, I made a mistake, I meant the BR ReflectionMethod @kukulich 🙄 sorry

Reply:
Reported in #596

Reply:
@asgrim I'm not sure. BR is "better" so it should maybe report only really `abstract` methods...

But there's `ReflectionMethod::getModifiers()` based also on `isAbstract()`...

You tell me how I should fix it :D

Reply:
I wasn't kidding when I said that PHPStan is an integration test suite for BetterReflection :D https://twitter.com/OndrejMirtes/status/1261621042176548865

Reply:
> You tell me how I should fix it :D

Oooooooooof

Guess we'll need more tests, but if you have a bug or scenario in mind, report a new issue referencing this thread.

The test suite and code are in good shape enough to "outsource" all of this work to the twitters :-)